test/test_retain_graph.py

```diff
@@ -0,0 +1,51 @@
+"""Test autograd functionality like retain_graph."""
+from pytest import raises
+from torch import rand, randint
+from torch.nn import CrossEntropyLoss, Linear, Module, Sequential
+
+from backpack import backpack, extend
+from backpack.extensions import BatchGrad
+
+
+def test_retain_graph():
+    model = extend(Sequential(Linear(4, 6), Linear(6, 5)))
+    loss_fn = extend(CrossEntropyLoss())
+
+    # after a forward pass graph is not clear
+    loss = loss_fn(model(rand(8, 4)), randint(5, (8,)))
```

Comment:
extract input and labels, fix random seed.

Reply:
Fixed the see.
What do you mean by "extract input and labels"?

Reply:
```
inputs = rand(8, 4)
labels = randint(5, (8,))
```
Then you can re-use them further below

Reply:
Ok, implemented it.